packages/server/src/integration-test/mysql.spec.ts

```diff
@@ -47,17 +47,13 @@ describe("mysql integrations", () => {
     makeRequest = generateMakeRequest(apiKey, true)
 
     mysqlDatasource = await config.api.datasource.create(
-      await databaseTestProviders.mysql.datasource()
+      await getDatasource(DatabaseName.MYSQL)
     )
   })
 
-  afterAll(async () => {
-    await databaseTestProviders.mysql.stop()
-  })
-
   beforeEach(async () => {
     primaryMySqlTable = await config.createTable({
-      name: uuidv4(),
+      name: generator.guid().replaceAll("-", "_").substring(0, 10),
```

Comment:
In cases like this, should we not use `generator.word({length})`?

Reply:
No, table names need to be unique, and `generator.word` is low-entropy.

Reply:
I did a comparison of this on one of Mel's PRs a little while ago: https://github.com/Budibase/budibase/pull/13165#discussion_r1510918899